apps/console/src/styles/app.css

```
@@ -1233,3 +1233,325 @@ table.records-table tbody tr.active td { background: transparent; }
+  gap: 8px;
+  margin-bottom: 8px;
+}
+
+.kg-detail-badges {
+  display: flex;
+  gap: 6px;
+  flex-wrap: wrap;
+}
+
+.kg-detail-title {
+  font-family: 'Lora', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Lora" (font-family-name-quotes)

(font-family-name-quotes)